lib/Pool.js

```diff
@@ -223,21 +223,22 @@ Pool.prototype._next = function () {
  */
 Pool.prototype._getWorker = function() {
   // find a non-busy worker
-  for (var i = 0, ii = this.workers.length; i < ii; i++) {
-    var worker = this.workers[i];
-    if (!worker.busy()) {
+  var workers = this.workers;
+  for (var i = 0; i < workers.length; i++) {
```

Comment:
Memoizing the array length ended up being counter productive, as I believe the internal bounds check was required anyways... Removing this removed a hot-spot the profiler raised.